package.json

```diff
@@ -92,11 +92,11 @@
     "packages/perf-benchmarks"
   ],
   "engines": {
-    "node": ">=10.13.0 <11.0.0",
-    "yarn": ">=1.10.1"
+    "node": ">=10.16.3 <10.17.0",
+    "yarn": ">=1.17.3 <1.18.0"
```

Comment:
```suggestion
    "yarn": "^1.17.3"
```

Reply:
No, we don't want to allow the minor version to float up higher than our spec.